tools/sql/main.go

```diff
@@ -38,12 +38,15 @@ func RunTool(args []string) error {
 }
 
 // root handler for all cli commands
-func cliHandler(c *cli.Context, handler func(c *cli.Context) error) {
-	quiet := c.GlobalBool(schema.CLIOptQuiet)
+func cliHandler(c *cli.Context, handler func(c *cli.Context) error) error {
 	err := handler(c)
-	if err != nil && !quiet {
-		os.Exit(1)
+	if err != nil {
+		if quiet := c.Bool(schema.CLIOptQuiet); quiet {
+			return fmt.Errorf("error executing command")
```

Comment:
@shijiesheng  Just checking, this will still exit code 1?

Reply:
oh, I just read the flag definition... that's... weird, definately not what I'd expect with that flag

Reply:
Yes, it's still returning exit 1. I'll fix it